Sources/Turf/Geometries/Polygon.swift

```diff
@@ -10,6 +10,15 @@ public struct Polygon: Equatable {
     public init(_ coordinates: [[CLLocationCoordinate2D]]) {
         self.coordinates = coordinates
     }
+    
+    public init(outerRing: Ring, innerRings: [Ring] = []) {
+        var ringCoordinates = [outerRing.coordinates]
+        for innerRing in innerRings {
+            ringCoordinates.append(innerRing.coordinates)
+        }
+        
+        self.coordinates = ringCoordinates
```

Comment:
Nit: I didn’t try this, but it should be possible to map directly to the coordinates thanks to the default argument value above:

```suggestion
        coordinates = ([outerRing] + innerRings).map { $0.coordinates }
```